### PR TITLE
Process Explorer Design

### PR DESCRIPTION
**Pull request by kruplm**
This PR contains the base design which mostly focuses on the visuals. 
It's going to be made more accurate by several user stories.

### CODE REVIEW

**Comment by bingenito** [comment]
Need to add license headers to source files

File: process explorer/src/app/master-view/processes/processes.component.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    [options]="chartOptions"
+    style="width: 100%; height: 400px; display: block;"
+  ></highcharts-chart>
+</div>
+<div class="column-layout_1 column">
+  <div class="row-layout row">
+  <igx-grid [data]="mockProcessesData" [rowSelection]= "'multiple'" [autoGenerate]="false" width="100%" height="607px" displayDensity="cosy" primaryKey="id" class="grid">
+    <igx-column field="id" header="id" title="id" dataType="string" [groupable]="true" [editable]="true"></igx-column>
+    <igx-column field="bits" header="bits" title="bits" dataType="string" [groupable]="true" [editable]="true"></igx-column>
+    <!--<igx-column field="services" header="services" title="services" dataType="number" [groupable]="true" [editable]="true"></igx-column>-->
```
Comment (bingenito) [inline process explorer/src/app/master-view/processes/processes.component.html:16]: Intentionally commented out?

File: process explorer/src/assets/GitHub/github.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32.58 31.77"><defs><style>.cls-1{fill:#191717;fill-rule:evenodd;}</style></defs><title>GitHub</title><g id="Layer_2" data-name="Layer 2"><g id="Layer_1-2" data-name="Layer 1"><path class="cls-1" d="M16.29,0a16.29,16.29,0,0,0-5.15,31.75c.82.15,1.11-.36,1.11-.79s0-1.41,0-2.77C7.7,29.18,6.74,26,6.74,26a4.36,4.36,0,0,0-1.81-2.39c-1.47-1,.12-1,.12-1a3.43,3.43,0,0,1,2.49,1.68,3.48,3.48,0,0,0,4.74,1.36,3.46,3.46,0,0,1,1-2.18c-3.62-.41-7.42-1.81-7.42-8a6.3,6.3,0,0,1,1.67-4.37,5.94,5.94,0,0,1,.16-4.31s1.37-.44,4.48,1.67a15.41,15.41,0,0,1,8.16,0c3.11-2.11,4.47-1.67,4.47-1.67A5.91,5.91,0,0,1,25,11.07a6.3,6.3,0,0,1,1.67,4.37c0,6.26-3.81,7.63-7.44,8a3.85,3.85,0,0,1,1.11,3c0,2.18,0,3.94,0,4.47s.29.94,1.12.78A16.29,16.29,0,0,0,16.29,0Z"/></g></g></svg>
```
Comment (bingenito) [inline process explorer/src/assets/GitHub/github.svg:1]: What is this?  Should it be part of source control?

File: process explorer/src/assets/indigo-design.svg [hunk 3]
```diff
@@ -0,0 +1,183 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (bingenito) [inline process explorer/src/assets/indigo-design.svg:1]: What is this?  Should it be part of source control?

File: process explorer/src/app/master-view/connections/connections.component.html [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+      <igx-icon font="material" [active]="true" class="icon_1">fiber_manual_record</igx-icon>
+      <igx-icon font="material" [active]="true" class="icon">fiber_manual_record</igx-icon>
+      <igx-icon font="material" [active]="true" class="icon_1">fiber_manual_record</igx-icon>
+      <igx-icon font="material" [active]="true" class="icon">fiber_manual_record</igx-icon>
+      <igx-icon font="material" [active]="true" class="icon_1">fiber_manual_record</igx-icon>
+      <igx-icon font="material" [active]="true" class="icon_1">fiber_manual_record</igx-icon>
+    </div>
+  </div>
+  <igx-grid [data]="mockConnectionsData" [allowFiltering]="true" filterMode="excelStyleFilter" [autoGenerate]="false" width="1052px" height="738px" displayDensity="cosy" primaryKey="title" class="grid">
+    <!-- <igx-column field="connected" header="connected" title="connected" dataType="boolean" [sortable]="true"></igx-column> -->
```
[the remark this answers is not in the document]
Comment (kruplm) [reply process explorer/src/app/master-view/connections/connections.component.html:25]: I intend to refine it with conditional formatting so it's gonna replace lines 11-22. But I remove this line for now.

File: process explorer/src/app/master-view/processes/processes.component.html (hunk 1 above)
Comment (kruplm) [reply process explorer/src/app/master-view/processes/processes.component.html:16]: Same with this.

**Comment by psmulovics** [comment]
> Need to add license headers to source files

So I think if that would be done, we would be good?

**Comment by kruplm** [comment]
> > Need to add license headers to source files
> 
> So I think if that would be done, we would be good?

What license do we want the project to fall under? GNU/GPL? Something else?

**Review by psmulovics — APPROVED** [review]

**Comment by psmulovics** [comment]
LGTM

**Pull request merged**